### PR TITLE
Fix bash quoting

### PR DESCRIPTION
**Pull request by ltshb**
The "Get Default Bump Type from PR and get PR Title" action did not worked on the following PR https://github.com/geoadmin/web-mapviewer/pull/985 due to bash word splitting.

In Bash to prevent globbing and word splitting you need to double quote the variables. So I corrected all bash variable to avoid such issues in future.

### CODE REVIEW

File: .github/workflows/pr-auto-semver.yml [hunk 1]
```diff
@@ -111,19 +113,19 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           PR_NUMBER: ${{ github.event.number }}
         run: |
-          gh api /repos/${{ github.repository }}/pulls/${PR_NUMBER} \
+          gh api "§/repos/${{ github.repository }}/pulls/${PR_NUMBER}" \
```
Comment (benschs) [inline .github/workflows/pr-auto-semver.yml:116]: I guess this is a typo? `§`

**Review by benschs — APPROVED** [review]
Cool, thanks!

File: .github/workflows/pr-auto-semver.yml (hunk 1 above)
Comment (ltshb) [reply .github/workflows/pr-auto-semver.yml:116]: oops very good catch

Comment (ltshb) [reply .github/workflows/pr-auto-semver.yml]: done

**Pull request merged**